src/components/RightPane/SectionTable/SectionTableBody.tsx

```diff
@@ -75,7 +76,12 @@ const NoPaddingTableCell = withStyles({
     sizeSmall: { padding: '0px 0px 0px 0px' },
 })(TableCell);
 
-const CourseCodeCell = withStyles(styles)((props) => {
+interface CourseCodeCellProps {
+    classes: ClassNameMap
+    sectionCode: string
+}
+
+const CourseCodeCell = withStyles(styles)((props: CourseCodeCellProps) => {
```

Comment:
Also in reference to this entire file, would it be better to split the individual cell components into multiple files? This file is starting to get quite long along with there being many different prop types. Maybe this PR isn't the right place to do these changes but just a thought.

Reply:
I don't think it's worth splitting because each of the cells is pretty small on its own so I don't think it would help. I do see your point about the file being big but I don't see any way of splitting it that would increase the readability.

Reply:
While I don't know for React components I feel like in general it's obviously best practice to compartmentalize code as much as possible. There is the downside that instead of having one long file we'll have a ton of small files but I feel like that isn't that much of a problem. I personally think it increases code readability because it would be much easier to figure out the main point of the file without having to read the entire thing.

Reply:
I'm going to leave it as-is for now. I started refactoring it and then realized that some of the styles are shared between the cells and the table body itself, and didn't want to abstract away the styles into different files because it splits up the component definitions, which I think we should avoid. If you can think of a way to get around that lmk and I'll throw it in the next PR in the TS conversion.

Reply:
Fair enough, I didn't look that deep into it. Considering that's how it works right now then leaving it as is is probably the better option.